app/controllers/feature_controller.rb

```diff
@@ -8,8 +8,22 @@ def toggle
     redirect_to root_path
   end
 
+  def full_record_toggle
+    session[:local_full_record] = session[:local_full_record].!
+    flash[:info] = "In app full record displays are now #{full_record_message}."
```

Comment:
I wonder whether this is the most human-friendly language ("on" vs "off" might be more expected). But maybe it doesn't matter for a thing we don't plan to deploy for super long.